Chartboost/ChartboostRouter.m

```diff
@@ -61,6 +62,12 @@ + (void)setDataUseConsentWithMopubConfiguration
 
 + (void)startWithParameters:(NSDictionary *)parameters completion:(void (^)(BOOL))completion
 {
+    if (SYSTEM_VERSION_LESS_THAN(kGADMAdapterMinimumOSVersion)) {
+        NSLog(@"Chartboost minimum supported OS version is iOS %@. Requested action is a no-op.", kGADMAdapterMinimumOSVersion);
```

Comment:
I don't see any NSLog in the project. I think it's better to stick to MPLogEvent to track issues, which is used everywhere.
We probably should define a new error in the NSError+ChartboostErrors.h for this